src/views/transaction-details/components/chain-row/chain-row.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from "react";
+
+import { ReactComponent as EthChainIcon } from "src/assets/icons/ethereum.svg";
+import { ReactComponent as HermezChainIcon } from "src/assets/icons/polygon-hermez.svg";
+import Typography from "src/views/shared/typography/typography.view";
+import useChainRowStyles from "src/views/transaction-details/components/chain-row/chain-row.style";
+
+interface ChainRowProps {
+  type: "l1" | "l2";
+  side: "to" | "from";
+  className?: string;
+}
+
+const ChainRow: FC<ChainRowProps> = ({ type, side, className }) => {
```

Comment:
I'd change the name of this component to just `Chain`.

It's used in a row by `transaction-details.view`, but that has nothing to do with the component.

Reply:
I agree with this as well

Reply:
Fixed.